test/CSVHelperTest.java

```diff
@@ -0,0 +1,50 @@
+import LunchManCore.Apprentice;
+import LunchManCore.FridayLunch;
+import org.junit.Test;
+import services.CSVHelper;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+public class CSVHelperTest {
+
+    @Test
+    public void createsApprenticesFromListOfNamesInCSV() throws Exception {
+        String mockApprenticesCSV = "/Users/molliestephenson/Java/LunchMan/test/mockApprentices.csv";
+        List<Apprentice> result = CSVHelper.createApprenticesFromCSV(new ArrayList<>(), mockApprenticesCSV);
+        assertEquals("Mollie", result.get(0).getName());
+        assertEquals("Nick", result.get(1).getName());
+        assertEquals("Ced", result.get(2).getName());
+        assertEquals("Priya", result.get(3).getName());
+        assertEquals("Rabea", result.get(4).getName());
+    }
+
+    @Test
+    public void createsRotaFromCSV() throws Exception {
+        String mockSchedule = "/Users/molliestephenson/Java/LunchMan/test/mockSchedule.csv";
```

Comment:
There are some JUnit Rules (TemporaryFolder) which allow you to create, and populate files for testing. One advantage is that you dont have to put in a path to that folder or file, and also, they are auto-cleaned up after. Might be worth having a look at to see if it would be advantageous over a committed file.